tests/integration/vpp_integration.sh

```diff
@@ -25,7 +25,7 @@ echo -e " VPP Integration Test - \e[1;33m${vppver}\e[0m"
 echo "============================================================="
 
 # run integration test
-if docker exec -i "$cid" /vpp-integration.test $*; then
+if docker exec -i "$cid" /vpp-integration.test "$*"; then
```

Comment:
Will this one really work with `$*` in quotes?

Reply:
yeah, without those quotes it works, but with them I get:
```sh
➜ VPP_IMG=ligato/vpp-base:19.08 ./tests/integration/vpp_integration.sh -test.run TestPing -test.v
=============================================================
 VPP Integration Test - 19.08.1-353~g418b22593~b80
=============================================================
flag provided but not defined: -test.run TestPing -test.v
Usage of /vpp-integration.test:
  -debug
        Turn on debug mode.
  -retry uint
        Number of VPP connect retries (default 3)
[...]
```

but it also works with `"$@"`, so actually I'm not sure why their autofix changes `$*` to `"$*"` and not to `"$@"`.